db/migrate/20240131202431_add_use_single_step_invite_and_approve_partner_process_to_organization.rb

```diff
@@ -0,0 +1,6 @@
+class AddUseSingleStepInviteAndApprovePartnerProcessToOrganization < ActiveRecord::Migration[7.0]
+  def change
+    add_column :organizations, :use_single_step_invite_and_approve_partner_process, :boolean
+    change_column_default :organizations, :use_single_step_invite_and_approve_partner_process, false
```

Comment:
Will be adding a rake task to update the current records to have default `false` and also a not null constraint.

Reply:
You should be able to do that within the migration.